### PR TITLE
Add mentorship resource in api spec

### PR DESCRIPTION
**Pull request by Echo2roy**

### CODE REVIEW

**Comment by kraulain** [comment]
Could you make it such that a mentor can have one or more mentees?

File: src/resources/api-specification/libraries/types/Mentee.raml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+properties:
+  id?: string
+  name: string
+  groovyExperience:
+   type: string
+   enum: [3 day bootcamp, multi-week course, 6 month course, up to 1 year experience at a job]
+   required: true
+  lib_frmwrk_tools:
+   type: string
+   enum: [grails, spring Boot, Ratpack, VertX, Spock, Google cloud Platform, AWS deployments, Cloud Foundry, Heroku,
```
Comment (jlstrater) [inline src/resources/api-specification/libraries/types/Mentee.raml]: Can you capitalize the G in Grails and the C in Google Cloud Platform? Or make all of the enum values lowercase?

**Review by jlstrater — COMMENTED** [review]
One small thing otherwise 👍

**Comment by kraulain** [comment]
I think it's preferable for you to add a **mentor**  field to the **mentee** object given that a mentee has one and only one mentor, rather than adding an array of mentees to the mentor object.

**Pull request merged**